### PR TITLE
Standardizing user experience with Excel snippets

### PR DESCRIPTION
**Pull request by AlexJerabek**
This is the first pass to ensure users experience all snippets in the same way. The CSS has been standardized, the button pattern is now similar, and the instructional text has been normalized. Several bugs or inaccurate descriptions were also fixed.

### CODE REVIEW

File: samples/excel/70-events/events-disable-events.yaml [hunk 1]
```diff
@@ -147,18 +147,27 @@ template:
         </section>
         <section class="samples ms-font-m">
             <h3>Try it out</h3>
+            <p>The handlers update the "Grand Total" cell when events are fired (and enabled). Try editing the cells or refreshing the data with events enabled amd disabled.</p>
```
Comment (mscharlock) [inline samples/excel/70-events/events-disable-events.yaml]:
```suggestion
            <p>The handlers update the "Grand Total" cell when events are fired (and enabled). Try editing the cells or refreshing the data with events enabled and disabled.</p>
```

File: samples/excel/89-preview-apis/range-areas.yaml [hunk 2]
```diff
@@ -150,15 +150,15 @@ template:
             <button id="color-all-logical-text-ranges" class="ms-Button">
                 <span class="ms-Button-label">Color logical and text ranges</span>
             </button>
-            <p>In Excel, select range <b>C3:C5</b> and format it's fill to a new color. Then set the fill for range <b>E3:E5</b> to <i>exactly the same color!</i> Then press <b>Read properties of <i>Qty</i> and <i>Unit Price</i> ranges</b>. Watch the console.</p>
+            <p>In Excel, select range <b>C3:C5</b> and format it's fill to a new color. Then set the fill for range <b>E3:E5</b> to <i>exactly the same color!</i> Then press "Read properties of <i>Qty</i> and <i>Unit Price</i> ranges". Watch the console.</p>
```
Comment (mscharlock) [inline samples/excel/89-preview-apis/range-areas.yaml]:
```suggestion
            <p>In Excel, select range <b>C3:C5</b> and format its fill to a new color. Then set the fill for range <b>E3:E5</b> to <i>exactly the same color!</i> Then press "Read properties of <i>Qty</i> and <i>Unit Price</i> ranges". Watch the console.</p>
```

File: samples/excel/37-workbook/protect-data-in-worksheet-and-workbook-structure.yaml [hunk 3]
```diff
@@ -179,7 +179,7 @@ script:
 template:
     content: |-
         <section class="ms-font-m">
-            <p>This sample shows how to protect 1. data in a worksheet and 2. the workbook's structure.</p>
+            <p>This sample shows how to protect data in a worksheet and the workbook's structure.</p>
```
Comment (ElizabethSamuel-MSFT) [inline samples/excel/37-workbook/protect-data-in-worksheet-and-workbook-structure.yaml]: maybe: "...protect a worksheet's data and the workbook structure."

**Comment by AlexJerabek** [comment]
Thanks for the feedback @mscharlock and @ElizabethSamuel-MSFT. Changes applied.

**Pull request merged**